src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,54 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+
+  @Test
+  @DisplayName("[요구사항1] Set의 size() 메소드를 활용해 numbers의 크기를 확인하는 테스트")
+  void setSizeTest() {
+    assertThat(numbers.size()).isEqualTo(3);
```

Comment:
size를 꺼내서 비교하는것말고 assertj 의 api 를 사용할수도 있어요!

```suggestion
    assertThat(numbers).hasSize(3);
```